src/pages/workspace/WorkspacesListPage.js

```diff
@@ -178,32 +180,60 @@ class WorkspacesListPage extends Component {
-                    onBackButtonPress={() => Navigation.navigate(ROUTES.SETTINGS)}
-                    onCloseButtonPress={() => Navigation.dismissModal(true)}
-                />
-                {_.isEmpty(workspaces) ? (
-                    <BlockingView
-                        icon={Expensicons.Building}
-                        title={this.props.translate('workspace.emptyWorkspace.title')}
-                        subtitle={this.props.translate('workspace.emptyWorkspace.subtitle')}
-                    />
-                ) : (
-                    <ScrollView style={styles.flex1}>{_.map(workspaces, (item, index) => this.getMenuItem(item, index))}</ScrollView>
-                )}
-                <FixedFooter style={[styles.flexGrow0]}>
+            <IllustratedHeaderPageLayout
+                title={this.props.translate('common.workspaces')}
+                shouldShowBackButton
+                onBackButtonPress={() => Navigation.navigate(ROUTES.SETTINGS)}
+                onCloseButtonPress={() => Navigation.dismissModal(true)}
+                illustration={WorkspacePlanetAnimation}
+                backgroundColor={themeColors.workspaceSettingsPageBackgroundColor}
+                footer={
                     <Button
                         accessibilityLabel={this.props.translate('workspace.new.newWorkspace')}
                         success
                         text={this.props.translate('workspace.new.newWorkspace')}
                         onPress={() => Policy.createWorkspace()}
                     />
-                </FixedFooter>
-            </ScreenWrapper>
+                }
+            >
+                {_.isEmpty(workspaces) && (
+                    <View style={[styles.flex1, styles.ph4]}>
+                        <Text style={[styles.textHeadline]}>{this.props.translate('workspace.emptyWorkspace.title')}</Text>
+                        <Text style={[styles.textBody, styles.mt2]}>{this.props.translate('workspace.emptyWorkspace.subtitle')}</Text>
+                        <View style={styles.workspacesListPageFeatureContainer(workspaceFeatures.length)}>
+                            {_.map(workspaceFeatures, ({icon, translationKey}) => (
+                                <View
+                                    style={[styles.flexRow, styles.alignItemsCenter]}
+                                    key={translationKey}
+                                >
+                                    <Icon
+                                        src={icon}
+                                        width={variables.iconSizeSuperLarge}
+                                        height={variables.iconSizeSuperLarge}
+                                    />
+                                    <Text style={[styles.h3, styles.pl2]}>{this.props.translate(translationKey)}</Text>
+                                </View>
+                            ))}
+                        </View>
+                    </View>
```

Comment:
Should we make these feature lists their own component? I think the illustration + feature pattern is likely to be repeated. Or should we save that refactor until it is repeated?